instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py

```diff
@@ -430,6 +431,18 @@ def traced_execution(
                     ):
                         commenter_data.update(**_get_opentelemetry_values())
 
+                    # Add flask related tags
+                    sqlcommenter_flask_values = (
+                        opentelemetry_context.get_value(
+                            "SQLCOMMENTER_FLASK_VALUES"
```

Comment:
Is it possible to move this framework specific things to commenter code and remove these references from here.